CHANGELOG_COSMOS.md

```diff
@@ -1,5 +1,10 @@
 # Unreleased
 
+# cosmoshub-4-v1.0.0 - 2022-09-03
+
+## Changes
+- Add light mode
```

Comment:
Please add pr number and link.

Another thing, putting changes under tag means we release changes with new tag. Just wanna double check since I don't see exisiting `cosmoshub-4-v1.0.0` in repo yet, did you planned to release this new tag after pr merge? Just a reminder putting changes under `#Unreleased` and `version tag` mean differently.

Reply:
Thanks @yayay927 , I have revised the change log https://github.com/forbole/big-dipper-2.0-cosmos/pull/989/files

Reply:
Hey @teamchong thanks for the change. However, the file still needs change. I'll explain with the following 2 possible situations.

If you're not going to release `cosmoshub-4-v1.0.0` tag, then there's no need for tag name and only need `Unreleased`. The file should look like

```
# Unreleased

## Changes
- Add light mode ([\#988](https://github.com/forbole/big-dipper-2.0-cosmos/issues/988))
- Merged `base-v2.1.0`
- Merged base
```


If you're going to release the tag after pr merge, then the file should look like
```
# Unreleased

## Changes
- 

# cosmoshub-4-v1.0.0 - 2022-09-03

## Changes
- Add light mode ([\#988](https://github.com/forbole/big-dipper-2.0-cosmos/issues/988))
- Merged `base-v2.1.0`
- Merged base
```

Hope this time explanation is more clear :) Lemme know if anything is unclear!

Reply:
updated